README.md

```diff
@@ -1373,3 +1374,9 @@ catch (e) {
   console.log(e);
 }
 ```
+
+## Set SipPeer Origination Settings
+
+```js
+
```

Comment:
Was this purposely left blank?

Reply:
nope. i'm dumb and forgot to put an example in there. updated